includes/class-newspack-newsletters-renderer.php

```diff
@@ -760,9 +955,35 @@ public static function mjml_api_credentials() {
 		return false;
 	}
 
+	/**
+	 * Expose one or more Rest routes for a public API.
+	 *
+	 * @return mixed
+	 */
+	public function register_newsletter_renderer_routes() {
+		$arguments = array(
+			/**
+			 * Return an array of properties required to render a useful ads warning.
+			 *
+			 * @uses Newspack_Newsletters_Renderer::get_ads_warning_in_editor()
+			 */
+			'callback' => array( $this, 'get_ads_warning_in_editor' ),
+			'methods'  => 'GET',
```

Comment:
A `permission_callback` key [is required in arguments since WP 5.5](https://developer.wordpress.org/reference/functions/register_rest_route/#changelog)